zeitschrift-fur-medienwissenschaft.csl

```diff
@@ -20,493 +14,297 @@
     <category field="humanities"/>
     <issn>1869-1722</issn>
     <summary>Style for zfm. Zeitschrift für Medienwissenschaft. Citations in notes; no bibliography.</summary>
-    <updated>2012-09-27T22:06:38+00:00</updated>
+    <updated>2014-09-13T22:01:31+00:00</updated>
     <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
   </info>
-  <!--
-================================================================================
-   TODO
-================================================================================
--->
-  <!--
-================================================================================
-   TERMS
-================================================================================
--->
+  <!-- HINWEISE
```

Comment:
Okay. Since it's quite a bit of text: In German or English? (German
seems more appropriate since the journal is published in German. But
then the first sentence would be in English and the rest in another
language.) Also: Is there a way to structure the text in "summary" (I
use a heading and bullet points for now)?

> On 2015-01-04T16:21:48+0100, Sebastian Karcher said:
>      <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>

>    </info>
> -  <!--
> -================================================================================
> -   TODO
> -================================================================================
> --->
> -  <!--
> -================================================================================
> -   TERMS
> -================================================================================
> --->
> +  <!-- HINWEISE

> not commented out, but instead encased in ```<summary>```
> ```</summary>```

> --- Reply to this email directly or view it on GitHub:
> https://github.com/citation-style-language/styles/pull/1313/files#r22439955

--
Till A. Heilmann, Dr. phil.

Online-Texte http://www.tillheilmann.info/german.php#OnlineTexts

Universität Siegen
Philosophische Fakultät
Medienwissenschaftliches Seminar
Adolf-Reichwein-Straße 2, Raum AR-K 611
57076 Siegen
Telefon: +49 271 740-4791
Telefax: +49 271 740-2731
www.uni-siegen.de/phil/medienwissenschaft/
www.multimediale-systeme.de